src/PerfView/PerfViewData.cs

```diff
@@ -493,6 +498,38 @@ public override void Open(Window parentWindow, StatusBar worker, Action doAfter
                     m_singletonStackSource.Viewer.Focus();
             }
         }
+
+        public void OpenOnExistingWorker(Action doAfter = null)
```

Comment:
It is not clear what OpenOnExistingWorker does and what the scenario is.  Can you describe why you need it?   I would like to avoid a new API if I can.  Also, generally speaking public opens methods (which are called from the GUI thread typically should not run 'OpenImpl' on that thread (that is what the status bar is for, you run it on the 'background' and the status bar lets the users kill if if he wants).   

My knee-jerk reaction is that 'OpenOnExisting' should not exist because 'Open' will just 'do the right thing' if it is already open (if that is not true, we should fix it so it is true).  

Comments?